### PR TITLE
[plugin.video.tviplayer] 0.2.0

### PR DESCRIPTION
**Pull request by darksky83**
* Bug fix for direrent flash player use in some Shows
* Change from strings.xml to strings.po

### CODE REVIEW

**Comment by Razzeee** [comment]
Should be fine.

File: plugin.video.tviplayer/resources/ondemand.py [hunk 1]
```diff
@@ -246,6 +240,7 @@ def tvi_resolver(url):
     try:
         source = abrir_url(url)
     except:
+        print "Unexpected error:", sys.exc_info()[0], sys.exc_info()[1]
```
Comment (learningit) [inline plugin.video.tviplayer/resources/ondemand.py]: Please remove print statement, use log() instead

File: plugin.video.tviplayer/resources/ondemand.py [hunk 2]
```diff
@@ -261,6 +256,7 @@ def tvi_resolver(url):
             if match[0][0]:
                 return match[0]
         except:
+            print "Unexpected error:", sys.exc_info()[0], sys.exc_info()[1]
```
Comment (learningit) [inline plugin.video.tviplayer/resources/ondemand.py]: Please remove print statement, use log() instead

**Comment by darksky83** [comment]
I will create a new PR whit the corrections.

**Pull request closed without merging**